bin/near-cli.js

```diff
@@ -160,6 +160,11 @@ yargs // eslint-disable-line
         type: 'string',
         default: config.nodeUrl
     })
+    .option('headers', {
```

Comment:
IMO, we don't need to allow users to define arbitrary header contents at the CLI level for an initial release of custom RPC URL and API key support; :hocho: this entirely until we need it IMHO

Reply:
This option is hidden and won't be shown to the users.